app/build.gradle

```diff
@@ -170,6 +170,13 @@ android {
     exclude "META-INF/LICENSE.txt"
     exclude "META-INF/NOTICE.txt"
   }
+    
+  lintOptions {
+        checkReleaseBuilds false
+        // Or, if you prefer, you can continue to check for errors in release builds,
+        // but continue the build even when errors are found:
+        abortOnError false
```

Comment:
why was this added?

Reply:
Got an build error and this was suggested in the error message to add to the gradle setting.
I'm using Android Studio 4.0 on Windows, all new installation.